custom-dockerfiles/cinder-volume/Dockerfile

```diff
@@ -4,7 +4,7 @@ USER root
 WORKDIR /tmp
 
 RUN sed -i 's/$stream/8-stream/g' /etc/yum.repos.d/CentOS-* && \
-        yum -y install git && \
+        yum -y install --nogpgcheck git && \
```

Comment:
`--nogpgcheck` was a workaround and we should figure out how to correctly configure yum so that we don't need this check.

Reply:
I think the correct way to resolve this is to import the rpm's GPG key to the proper location and then run `yum install git` without `--nogpgcheck`